app/src/main/res/drawable/ic_watch_later_black_24dp.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#FF000000"
```

Comment:
Just nitpicking here, but can you make this white and rename the file (since the other icons are white)? It doesn't really affect anything since it's tinted, but it's just for consistency.